system/modules/core/library/Contao/File.php

```diff
@@ -316,6 +306,59 @@ public function __get($strKey)
 				return $this->imageSize[1];
 				break;
 
+			case 'imageViewSize':
+				if (empty($this->arrImageViewSize))
+				{
+					if ($this->imageSize) {
+						$this->arrImageViewSize = array
+						(
+							$this->imageSize[0],
+							$this->imageSize[1]
+						);
+					}
+					elseif ($this->isSvgImage)
+					{
+						$doc = new \DOMDocument();
+
+						if ($this->extension == 'svgz')
+						{
+							$doc->loadXML(gzdecode($this->getContent()));
+						}
+						else
+						{
+							$doc->loadXML($this->getContent());
+						}
+
+						$svgElement = $doc->documentElement;
+
+						if ($svgElement->getAttribute('viewBox'))
+						{
+							$svgViewBox = preg_split('/[\s,]+/', $svgElement->getAttribute('viewBox'));
+
+							$this->arrImageViewSize = array
+							(
+								intval($svgViewBox[2]),
```

Comment:
The `viewBox` attribute only allows unitless numbers, so `Image::getPixelValue()` should not be used here.